components/web_discovery/browser/web_discovery_service.cc

```diff
@@ -65,6 +68,7 @@ void WebDiscoveryService::RegisterProfilePrefs(PrefRegistrySimple* registry) {
 void WebDiscoveryService::Shutdown() {
   Stop();
   pref_change_registrar_.RemoveAll();
+  backup_results_service_ = nullptr;
```

Comment:
not much sense in this, b/c it's raw pointer and you don't do anything in the destructor anyway